bin/ci/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -o xtrace
+set -o errexit
+
+# The build system. Either 'Unix Makefiles' or 'Ninja'.
+GENERATOR=${GENERATOR:-Unix Makefiles}
+# The compiler. Either 'gcc' or 'clang'.
+COMPILER=${COMPILER:-gcc}
+# The build type. Either 'Debug' or 'Release'.
+BUILD_TYPE=${BUILD_TYPE:-Debug}
+# Additional arguments to CMake.
+CMAKE_ARGS=${CMAKE_ARGS:-}
```

Comment:
for CI builds I think we want `-Dwerr=ON` always...I guess it could be added here or somewhere in the yaml config

Reply:
I'll set that as the default with `${CMAKE_ARGS-'-Dwerr=ON'}` substitution. That way callers can remove it by setting an empty `$CMAKE_ARGS`.